src/components/ModalPostVote.vue

```diff
@@ -80,6 +85,14 @@ function share(shareTo: 'twitter' | 'lenster') {
           {{ $t('shareOnLenster') }}
         </BaseButton>
 
+        <BaseButton
```

Comment:
I'm already subscribed and I still see the "Subscribe" button after voting 
<img width="634" alt="image" src="https://github.com/snapshot-labs/snapshot/assets/51686767/1b8dd183-13b7-4c0e-b3d0-cb8102bd82a9">